examples/src/keyring/aws_kms/act_like_aws_kms_master_key_provider.py

```diff
@@ -0,0 +1,88 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+In earlier versions of the AWS Encryption SDK,
+you used master key providers to determine how your data keys are protected.
```

Comment:
```suggestion
You might have used master key providers to protect your data keys in an earlier versions of the AWS Encryption SDK. This example shows how to replace a master key provider with a keyring that behaves the same way.
```
I might never have used the ESDK before.